Cargo.toml

```diff
@@ -18,11 +18,12 @@ geo-types ="0.6.0"
 libc = "0.2.62"
 num-traits = "0.2.8"
 thiserror = "1.0.4"
-reqwest = { version = "0.10.6", default-features= false, features = ["blocking", "rustls-tls"] }
+reqwest = { version = "0.10.6", optional = true, default-features = false, features = ["blocking", "rustls-tls"] }
 
 [features]
 bundled_proj = [ "proj-sys/bundled_proj" ]
```

Comment:
For my own understanding, I think "bundled_proj" means "I have externally installed proj, do not build proj as part of proj-sys". Is that right?

Reply:
No, the other way around! bundled_proj builds libproj from the vendored source as part of the proj-sys build.